content/blog/2023/08/28/2023-08-28-number-of-open-issues-probe.adoc

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+title: Number of Open Issues Probe
+tags:
+- gsoc
+- gsoc2023
+- healthscore
+- probes
+- plugin
+authors:
+- jagruti
+opengraph:
+  image: /images/gsoc/2023/jagruti/number-of-open-issues-probe.png
+sig:
+- gsoc
+---
+
+== Introduction
+
+An alarming number of issues in a plugin may signify that the plugin is outdated, in need of a new maintainer, or in need of a complete revamp.
+Open issues might also mean that there are a lot of new feature requests, which in turn means the plugin has quite a number of active users.
+The number of open issue probe counts the number of open issues in the issue tackers.
+
+NOTE: As of now, the probe only counts open issues in GitHub and the JIRA issue tacker.
+
+== Importance of the probe
+
+The goal of the probe is to help maintainers and contributors identify plugins they want to invest their time into.
+This would lead to maintainers keeping plugins updated and useful for the users.
+
+== Tasks and Challenges
+
+During the Google Summer of Code project duration, I was intended to:
+
+- Figure out the correct class design.
+- Understand the Update Center and how it works.
+- Implement the link:https://github-api.kohsuke.org/[GitHub] and link:https://developer.atlassian.com/server/jira/platform/rest-apis/[JIRA] APIs.
+
+== Outcome
+
+The probe was approved and ready to be merged.
+
+== Learning
+
+Over the course of the project, I learned about the correct usage and implementation of the `abstract` class.
+This was a more complex version of the class I was used to.
```

Comment:
Really not sure about the wording here? Correct usage of the `abstract` class? I think it would have deserved a bit more context to explain what you really learned here?